lib/beaker-pe/install/pe_utils.rb

```diff
@@ -192,6 +192,11 @@ def installer_cmd(host, opts)
           end
         end
 
+        #This calls the installer command on the host in question
+        def execute_installer_cmd(host, opts)
```

Comment:
Since this helper is being added, I would add another commit to generalize its use elsewhere (scanning it looked like we use 'on host, installer_cmd(host, opts)' several other places.  Occasional the variable name changes... (host/master, for example), but it seems to otherwise be the same.

The other way of looking at this though is, do we gain anything by having this helper instead of just using 'on host, installer_cmd(host, opts)'?

Reply:
This method makes the spec tests much cleaner.

Reply:
Then I'd be in favor of using execute_installer_cmd generally.

Reply:
@jpartlow I'm satisfied with the usage of `execute_installer_cmd` here and elsewhere in the code. Is it good with you? I'm ready to merge this otherwise.

Reply:
Josh is out until the New Year, so as they say in the Planeteers, the power... is yours @tvpartytonight .